src/spring-cloud/azext_spring_cloud/buildpacks_binding.py

```diff
@@ -0,0 +1,55 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+# pylint: disable=wrong-import-order
+from ._enterprise import DEFAULT_BUILD_SERVICE_NAME
+from .vendored_sdks.appplatform.v2022_05_01_preview import models
+from azure.core.exceptions import ResourceNotFoundError
+from knack.util import CLIError
+
+
+def buildpacks_binding_create(cmd, client, resource_group, service,
+                              name, type, properties=None, secrets=None):
+    return _create_or_update_buildpacks_binding(client, resource_group, service, name,
+                                                type, properties, secrets)
+
+
+def buildpacks_binding_set(cmd, client, resource_group, service,
+                           name, type, properties=None, secrets=None):
+    return _create_or_update_buildpacks_binding(client, resource_group, service, name,
+                                                type, properties, secrets)
+
+
+def buildpacks_binding_show(cmd, client, resource_group, service, name):
+    try:
+        return client.buildpacks_binding.get(resource_group, service, DEFAULT_BUILD_SERVICE_NAME, name)
+    except ResourceNotFoundError as e:
+        raise CLIError("Buildpacks Binding {} is not existed in resource group {}, service {}"
```

Comment:
Can skip the customize error message here. As CLI core will handle this.

Reply:
Removed.

And yeah, CLI core will handle this error by print the error reason of exception, but it's too simple. However, it should be RP's fault.
```
(CLIDev0617) PS C:\devops\CLIEnvs\CLIDev0617> az spring-cloud build-service buildpacks-binding show -n my-binding -g jiec-enterprise --service jiec-e-asc
BuildpacksBinding was not found
```

Reply:
It is ARM's behaviro